src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -465,6 +465,7 @@ public static Configuration loadConfiguration(InputSource configSource,
      * @throws CheckstyleException if the string contains an opening
      *                           {@code ${} without a closing
      *                           {@code }}
+     * @noinspection MethodWithMultipleReturnPoints
```

Comment:
this suppression is not allowed.
Please fix the code.
It has 3 returns, 1st will be ignored as it is guard, 2nd and 3rd have to be fixed. In this method should be two return points in result.